pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,826 @@
+    "    syncOptions:\n",
+    "    - Validate=false\n",
+    "    - ApplyOutOfSyncOnly=true\n",
+    "\"\"\" % (CLUSTER_NAME, CLUSTER_REGION, CLUSTER_NAME)\n",
+    "%store text_input >app-of-apps/app-of-apps-\"$CLUSTER_NAME\".yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'app-of-apps/app-of-apps-my-cluster.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "List the file in `app-of-apps/kustomization.yaml`\n",
+    "```\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "namespace: argocd\n",
+    "resources:\n",
+    "  - app-of-apps-my-cluster.yaml\n",
+    "```\n",
+    "\n",
+    "#### Create an environment for cluster in ArgoCD\n",
+    "\n",
+    "As a next part we have to create environment in ArgoCD where we define resources for cluster.\n",
+    "Create a new folder for resources\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "source": [
+    "!mkdir  -p envs/$CLUSTER_REGION/$CLUSTER_NAME/cluster-management\n",
+    "text_input=\"\"\"\\\n",
+    "apiVersion: argoproj.io/v1alpha1\n",
+    "kind: Application\n",
+    "metadata:\n",
+    "  name: cluster-resources\n",
+    "spec:\n",
+    "  destination:\n",
+    "    name: %s\n",
+    "    namespace: open-cluster-management-agent\n",
+    "  ignoreDifferences:\n",
+    "    - group: imageregistry.operator.openshift.io\n",
+    "      jsonPointers:\n",
+    "        - /spec/defaultRoute\n",
+    "        - /spec/httpSecret\n",
+    "        - /spec/proxy\n",
+    "        - /spec/requests\n",
+    "        - /spec/rolloutStrategy\n",
+    "      kind: Config\n",
+    "      name: cluster\n",
+    "  project: cluster-management\n",
+    "  source:\n",
+    "    path: cluster-scope/overlays/prod/%s/%s\n",
+    "    repoURL: https://github.com/operate-first/apps.git\n",
+    "    targetRevision: HEAD\n",
+    "  syncPolicy:\n",
+    "    automated:\n",
+    "      prune: true\n",
+    "      selfHeal: true\n",
+    "    syncOptions:\n",
+    "    - Validate=false\n",
+    "    - ApplyOutOfSyncOnly=true\n",
+    "\n",
+    "\"\"\" % (CLUSTER_NAME, CLUSTER_REGION, CLUSTER_NAME)\n",
+    "%store text_input >envs/$CLUSTER_REGION/$CLUSTER_NAME/cluster-management/cluster-resources.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'envs/moc/my-cluster/cluster-management/cluster-resources.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Create `cluster-resources.yaml` in ` envs/$CLUSTER_REGION/$CLUSTER_NAME/cluster-management/`."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Create `kustomization.yaml` which lists `cluster-resources.yaml`"
```

Comment:
```suggestion
    "Since we use Kustomize for ArgoCD applications as well, we need to create a `kustomization.yaml` which lists `cluster-resources.yaml` here."
```

Reply:
Resolved in latest commit.